docs/GETTING_STARTED.md

```diff
@@ -132,15 +132,16 @@ does not have access to.
 
 ### Creating the SOCI index
 
-To create the index run:
+To create the index, which can be pushed to ORAS-compatible registry, run:
 ```
-sudo ./soci create localhost:5000/rabbitmq:latest
+sudo ./soci create --oras localhost:5000/rabbitmq:latest
 ```
-Behind the scenes SOCI is interacting with containerd.  So the tag
-`localhost:5000/rabbitmq` has to exist in containerd - that is what SOCI is
-referencing.  This created two kinds of objects.  The first is a series of ztocs
-(one per layer).  A ztoc is a table of contents for compressed data (z is the
-compression, toc is Table Of Contents).  The other is a manifest that relates
+The `--oras` flag instructs the CLI to build an ORAS-compatible manifest, 
+as opposed to an OCI Artifact Manifest. By default, the CLI will build the latter.
+Behind the scenes SOCI is interacting with containerd. So the tag `localhost:5000/rabbitmq` 
+has to exist in containerd - that is what SOCI is referencing.  This created two kinds of objects.  
```

Comment:
Punctuation nit: Semicolon (";") instead of dash ("-"). But really, I would just delete "- that is what SOCI is referencing".

Reply:
Done.